.storybook/preview-head.html

```diff
@@ -0,0 +1,59 @@
+<script>
+  (function() {
+   /**
+     * Searches for an element with a data attribute prefixed with "data-storybook-htmlattr-", and ending in the
+     * name of the desired attribute. If it finds the data attribute, it returns its value, or an empty string if
+     * it doesn't have a value, or the value is empty.
+     *
+     * @param {string} htmlAttributeName
+     * @return {string} - The value found to set for htmlAttributeName
+     */
+    const findValue = function(htmlAttributeName) {
+
+      const strWithInitialCapital = function(str) {
+        return str.charAt(0).toUpperCase() + str.substring(1);
+      };
+
+      const attributeSource = document.querySelector('[data-storybook-htmlattr-' + htmlAttributeName + ']');
+      if (attributeSource) {
+        return attributeSource.dataset['storybookHtmlattr' + strWithInitialCapital(htmlAttributeName)] || '';
+      }
+      return '';
+    };
```

Comment:
Shame DOM XPath isn't in IE.

Reply:
Yeah, writing this was a walk down memory lane.